db/migrate_cards/20151117121317_migrate_logo_to_image.rb

```diff
@@ -0,0 +1,50 @@
+# -*- encoding : utf-8 -*-
+
+class MigrateLogoToImage < Card::Migration
+  def up
+    # delete cards with logo and image
+    # change name from +logo to +image
+    # update some structure cards
+
+    company_image_structure_name = 'Company+image+*type plus right+*structure'
+    puts "delete #{company_image_structure_name}"
+    if (company_image_structure_card = Card[company_image_structure_name])
+      company_image_structure_card.delete
+    end
+    puts "delete #{company_image_structure_name}"
+    user_image_structure_name = 'User+image+*type plus right+*structure'
+    if (user_image_structure_card = Card[user_image_structure_name])
+      user_image_structure_card.delete
+    end
+    puts 'delete cards with logo and image'
+    cards_with_both = Card.search right_plus: %w(and logo image)
+    puts "\tfound #{cards_with_both.size}"
+    cards_with_both.each do |c|
+      card_name = "#{c.name}+logo"
+      puts "\tdeleting #{card_name}"
+      Card[card_name].delete
+    end
+    puts 'renaming card+logo to card+image and update the type id'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.